aaf-mdqt.gemspec

```diff
@@ -23,17 +23,17 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency 'commander', "~>4.6"
-  spec.add_dependency 'faraday', "~>2.9.0"
-  spec.add_dependency 'faraday-http-cache', "~>2.5.0"
-  spec.add_dependency 'faraday-follow_redirects', "~>0.3.0"
-  spec.add_dependency 'httpx', "~>1.2.0"
+  spec.add_dependency 'commander', "~>4"
+  spec.add_dependency 'faraday', "~>2"
+  spec.add_dependency 'faraday-http-cache', "~>2"
+  spec.add_dependency 'faraday-follow_redirects', "~>0.3"
+  spec.add_dependency 'httpx', "~>1"
   spec.add_dependency 'activesupport', "~>7"
```

Comment:
Rails 8 is coming. I'd just use `>` requirements for all these things

Reply:
ruby gems doesn't recommend > matchers the only reason i did it